plugins/auth/token.js

```diff
@@ -16,7 +16,7 @@ module.exports = () => ({
         notes: 'Generate a JWT for use throughout Screwdriver',
         tags: ['api', 'auth', 'token'],
         auth: {
-            strategies: ['token', 'session', 'auth_token'],
+            strategies: ['session', 'auth_token'],
```

Comment:
I would have thought it only went through github if using the `oath` strategy.

My naive attempt of just making a `GET` request with an Authorization header borrowed from another request didn't work though, so I may be wrong.